nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
```

Comment:
- [ ] [OPTIONAL] It is always nice to make your error specific with messages i.e `raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`